config/environments/development.rb

```diff
@@ -75,7 +75,7 @@
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  config.three_scale.payments.enabled = false
+  config.three_scale.payments.enabled = true
```

Comment:
How will we handle this for SaaS? Does this impact all customers' stripe edit form transparently? Will they need to apply some change manually?

Reply:
We do not need to change this for SaaS. In SaaS, it is already true. And for development, we do not need it anymore since https://github.com/3scale/porta/pull/2309
So you won't even see this file changing in this PR when it is ready 😄

Reply:
Apologies, I see now this is _development.rb_ 🤦